privaterepo/ssh_user_list.go

```diff
@@ -0,0 +1,83 @@
+package privaterepo
+
+import (
+	"bytes"
+	"html/template"
+	"net/url"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/valuemodifier/path"
+)
+
+func ContentToSSHUserList(content string) (string, error) {
+	var err error
+
+	var rendered string
+	{
+		tmpl, err := template.New("main").Parse(content)
+		if err != nil {
+			return "", microerror.Mask(err)
+		}
+
+		v := struct {
+			AWS struct {
+				Region      string
+				HostCluster struct {
+					Account          string
+					AdminRoleARN     string
```

Comment:
Is it okay to have information like this possibly in our build logs? We store it in a configmap but having it accidentally exposed in public build logs seems like a step above that.

Reply:
I don't follow. This is not logged at all. This is only used to render the template so we can access the data we are interested in.

Reply:
It is not logged, I am just wondering how bad it would be if someone were to log it accidentally.

Reply:
Buddy, this is only the data structure used to render the template. There is no value given for it. It is only an empty string.

Reply:
I see the error in my ways here. Still curious how carefully we should treat this data (when it's in eventually).